admin_guide/diagnostics_tool.adoc

```diff
@@ -404,15 +415,18 @@ that your original SSH key file will be relabeled to something like
 `system_u:object_r:container_file_t:s0:c113,c247`. For more details about `:Z`,
 see the `docker-run(1)` man page.
 
-Keep this in mind for these volume mount specifications ￼because it could have
-unexpected consequences. For example, if you mount (and ￼therefore relabel) your
-*_$HOME/.ssh_* directory, *sshd* will become unable to access your public keys
-to allow remote login. To avoid altering the original file labels, mounting a
-copy of the SSH key (or directory) is recommended.
+[IMPORTANT]
+====
+These volume mount specifications could have unexpected consequences. For
+example, if you mount (and ￼therefore relabel) the *_$HOME/.ssh_* directory,
+*sshd* becomes unable to access the public keys to allow remote login. To avoid
+altering the original file labels, mounting a copy of the SSH key (or directory)
```

Comment:
s/mounting a copy of the SSH key (or directory) is recommended./ mount a copy of the SSH key or directory.